server/routes/getGlobalStatistics.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable guard-for-in */
+import { Request, Response, NextFunction } from 'express';
+import { factory, formatFilename } from '../../shared/logging';
+
+
+const getGlobalStatistics = async (models, req: Request, res: Response, next: NextFunction) => {
+    let result: any = [];
+    result = await models.HistoricalValidatorStats.findAndCountAll({
```

Comment:
we shouldn't be grabbing EVERY record in this DB, only the latest record for each active validator afaik. This is because we are constantly adding new records to this DB for every single event that causes a statistic to change. If we end up grabbing all of them, this will be highly expensive and could end up being thousands of events.

Reply:
Hi Drew.I have kept a flag 'isLatest' which would be set  false whenever new data would be inserted against any record.So I will fetch the records only which have 'isLatest' flag true.

Reply:
Makes sense! Comments would be helpful :)